lib/ruby/lib/alces/stack/templater.rb

```diff
@@ -203,14 +203,18 @@ def initialize(msg="Hash included through file method. Must be included in Combi
       class Finder
         def initialize(default_location, template)
           @default_location = default_location
-          @template = find_template(template)
-          @filename_ext = @template.scan(/\.?[\w\_\-]+\.?[\w\_\-]*\Z/)[0].chomp
-          @filename = @filename_ext.scan(/\.?[\w\_\-]+/)[0].chomp
+          @template = find_template(template).chomp
+          @filename_ext = File.basename(@template)
+          @filename_ext_trim_erb = @filename_ext.gsub(".erb", "")
```

Comment:
Unless it doesn't end in `.erb`; not sure what this value looks like at runtime.